packages/radix-ui-themes/src/components/text-area.css

```diff
@@ -27,7 +48,8 @@
   padding: var(--text-area-padding-y) var(--text-area-padding-x);
 
   &:where(:focus) {
-    outline: 2px solid var(--color-focus-root);
+    outline: 1px solid;
+    outline-color: inherit;
```

Comment:
Ended up having to maintain a focus outline both on the wrapper and on the actual text area input element

- Since our text area element comes in a wrapper div for layout reasons, we have to put `resize` on the actual div so that resizing horizontally works as expected
- To make `resize` work on the div, it needs `overflow: hidden`
- `overflow: hidden` trims the outer part of the focus outline of the text area input element
- Because of that, the focus outline needs to go on the wrapper
- However, `.rt-TextAreaChrome` box shadow goes on top of the inner part of the focus outline, hence we need to do the inner focus outline separately 

To visualize these problems...

2px outline just on the wrapper:

![Screenshot 2024-01-29 at 10 10 47](https://github.com/radix-ui/themes/assets/8441036/2c30c0c2-f122-4ea5-ab4c-e4e1fa1c8b73)

1px outer outline on the wrapper and 1px inner outline on the text area element

![Screenshot 2024-01-29 at 10 10 17](https://github.com/radix-ui/themes/assets/8441036/13baca67-28ac-4650-aced-670cacf7346c)

2px outline on the wrapper and 1px inner outline on the text area element (this version)

![Screenshot 2024-01-29 at 10 10 43](https://github.com/radix-ui/themes/assets/8441036/c46fe214-c258-4079-b3e3-a5f29e126916)

***

I think that we'll be able to shed a good part of the text area and text input CSS when the usage of Firefox versions that don't support `:has` fades away a bit more, for now this should be a good practical version.